New package: EEGToolkit v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: EEGToolkit
- Repository: https://github.com/slopezpereyra/EEGToolkit.jl
- Created by: @slopezpereyra
- Version: v0.1.0
- Commit: d21488fa65f69b56885d1de0cd3a8d0b866b8e35
- Git reference: HEAD
- Description: A Julia package for computational EEG analysis ~ With special tools for sleep neuroscience!

goerz commented:
[noblock] @slopezpereyra This looks good for the most part, except for still missing the compat bound for `DSP`. You can just add that in your `Project.toml`, and retrigger the registration once more. Do not change the version number. That way, the registration will update this existing PR instead of creating a new one.

JuliaHubRegistrator committed:
New package: EEGToolkit v0.1.0

UUID: 60d25398-4b8b-433c-a8d4-4fe867135cfd
Repo: https://github.com/slopezpereyra/EEGToolkit.jl.git
Tree: 87f20c64e0d82f3b57091ed594a5730b028953f4

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09

slopezpereyra commented:
Thanks @goerz, I did what you suggested.

goerz commented:
[noblock] Great, looks like it's good to go, and will auto-merge after the three-day waiting period ends. Make sure to edit [your comment](https://github.com/JuliaRegistries/General/pull/107928#issuecomment-213941282) to include `[noblock]`, though, or it will block the merge.

slopezpereyra commented:
Done! Thanks for all the assistance.
